### PR TITLE
[Feat] Home design

### PR DESCRIPTION
**Pull request by Denis-Dinkov**

### CODE REVIEW

File: src/views/home/index.tsx [hunk 1]
```diff
@@ -21,90 +22,41 @@ const Home = () => {
...
+              />
+              <Icon
+                name={card.iconName}
+                className="mb-7 text-dev-pink-500"
+                size={[40]}
+              />
+              <h4 className="mb-2 font-h4-bold">
+                {card.title}
+              </h4>
+              <p className="font-geist text-dev-black-300 font-body2-regular dark:text-dev-purple-300">
```
Comment (DimitarTAtanasov) [inline src/views/home/index.tsx:55]: let's move className on new line

File: src/views/home/index.tsx [hunk 2]
```diff
@@ -21,90 +22,41 @@ const Home = () => {
...
-            </h4>
-            <p className="font-geist text-dev-black-300 font-body2-regular dark:text-dev-purple-300">
-              Dive into our advanced developer console for seamless on-chain interaction and rapid prototyping. Experience a streamlined, user-friendly environment designed to accelerate your development process and boost productivity.
-            </p>
-          </PDLink>
+            >
+              <Icon
+                name="icon-linkArrow"
+                size={[24]}
+                className="absolute right-4 top-4 transition-transform duration-200 ease-linear group-hover:-translate-y-1 group-hover:translate-x-1"
```
Comment (DimitarTAtanasov) [inline src/views/home/index.tsx:45]: could we combine the props

File: src/constants/homeLinks.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+export const HOME_LINKS = [
```
Comment (DimitarTAtanasov) [inline src/constants/homeLinks.ts:1]: this could be in ..../constants/links.ts in my opinion

**Review by DimitarTAtanasov — CHANGES_REQUESTED** [review]

File: src/views/home/index.tsx [hunk 4]
```diff
@@ -21,90 +22,41 @@ const Home = () => {
...
-              Developer Console
-            </h4>
-            <p className="font-geist text-dev-black-300 font-body2-regular dark:text-dev-purple-300">
-              Dive into our advanced developer console for seamless on-chain interaction and rapid prototyping. Experience a streamlined, user-friendly environment designed to accelerate your development process and boost productivity.
-            </p>
-          </PDLink>
+            >
+              <Icon
+                name="icon-linkArrow"
+                size={[24]}
```
Comment (pivanov) [inline src/views/home/index.tsx:44]: 24 is the default value of <Icon /> component from a while (we can check the hole project and make a clean up)

File: src/assets/images/icons/icon/explorer.svg
Comment (pivanov) [inline src/assets/images/icons/icon/explorer.svg:1]: same

File: src/assets/images/icons/icon/forks.svg
Comment (pivanov) [inline src/assets/images/icons/icon/forks.svg:1]: same

File: src/assets/images/icons/icon/rpcCalls.svg
Comment (pivanov) [inline src/assets/images/icons/icon/rpcCalls.svg:1]: same

File: src/assets/images/icons/icon/chain.svg
Comment (pivanov) [inline src/assets/images/icons/icon/chain.svg:1]: we should export the svg without masks ... and reduce the groups <g />

File: src/assets/images/icons/icon/runtimeCalls.svg
Comment (pivanov) [inline src/assets/images/icons/icon/runtimeCalls.svg:1]: same

File: src/constants/homeLinks.ts (hunk 3 above)
Comment (pivanov) [reply src/constants/homeLinks.ts:1]: +1

**Review by DimitarTAtanasov — APPROVED** [review]

**Pull request merged**